browser/ntp_background/new_tab_takeover_infobar_delegate_unittest.cc

```diff
@@ -30,32 +37,160 @@ class NewTabTakeoverInfoBarDelegateTest
     return infobars::ContentInfoBarManager::FromWebContents(web_contents());
   }
 
+  PrefService* GetPrefs() { return profile()->GetPrefs(); }
+
+  void CreateInfobar() {
+    EXPECT_THAT(GetInfobarManager()->infobars().size(), ::testing::Eq(0));
+    NewTabTakeoverInfoBarDelegate::MaybeShowAndRecordInfobarShown(
+        web_contents(), GetPrefs());
+    EXPECT_THAT(GetInfobarManager()->infobars().size(), ::testing::Eq(1));
+  }
+
+  void CloseInfobar() {
+    GetInfobarManager()->RemoveAllInfoBars(false);
+    EXPECT_THAT(GetInfobarManager()->infobars().size(), ::testing::Eq(0));
```

Comment:
I believe `::testing::IsEmpty()` has support for `.size()`

Reply:
Fixed